backend/editor/entries.py

```diff
@@ -206,7 +225,13 @@ def update_node_children(entry, new_children_ids):
     existing_ids = [record['child.id'] for record in get_current_transaction().run(query, ids=list(added_children))]
     to_create = added_children - set(existing_ids)
 
-    for child in to_create:
+    #Normalising new children node ID

```

Comment:
pep8 ask for space after the `#`
```suggestion
    # Normalising new children node ID

```